scripts/demangle.py

```diff
@@ -2,23 +2,38 @@
 import json
 import glob
 import re
+import docker
 import ctypes
+import cpp_demangle
 
-json_path = '../dump/augeas'
-proj_path = '../oss-fuzz/projects/augeas'
-file_names = [f for f in os.listdir(json_path) if f.endswith('.json') and os.path.isfile(json_path+'/'+f)]
-print('Looking for json file: ',file_names)
-def extract_func_code(func_name, file_path):
-    with open(file_path, 'r') as file:
-        code_content = file.read()
+proj_name = 'tmux'
+json_path = '../dump/'+proj_name
+json_file_names = [f for f in os.listdir(json_path) if f.endswith('.json')]
+print('Looking for json file: ',json_file_names)
+cnt=0
+
+def extract_func_code(func_name, code_content, if_c_code=True):
```

Comment:
Try to `import typing` and add the types of the argument.

Reply:
Added types of the argument and I use argparse for parameter inputs.